app/views/layouts/_navbar.html.erb

```diff
@@ -1,5 +1,5 @@
 <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-  <a class="navbar-brand" href="#">MapApp</a>
+  <a class="navbar-brand" href="/">MapApp</a>
```

Comment:
Instead of using `/` user `link_to` with `root_path`. You will be pretty sure that always redirect to main path.